pkg/services/correlations/models.go

```diff
@@ -42,21 +49,31 @@ type CorrelationConfig struct {
 	// required:true
 	// example: { "expr": "job=app" }
 	Target map[string]interface{} `json:"target" binding:"Required"`
+	// Source data transformations
+	// required:false
+	// example: [{"type": "logfmt"}]
+	Transformations []Transformation `json:"transformations,omitempty"`
 }
 
 func (c CorrelationConfig) MarshalJSON() ([]byte, error) {
 	target := c.Target
+	transformations := c.Transformations
 	if target == nil {
 		target = map[string]interface{}{}
 	}
+	if transformations == nil {
```

Comment:
Is this `if` doing anything?

Reply:
Ah no, I added it when looking at the pattern for target, but then changed it where it no longer makes sense.